pyipmi/chassis.py

```diff
@@ -178,7 +178,11 @@ def get_system_boot_options(self, parameter_selector=0,
         req.set_selector = set_selector
         req.block_selector = block_selector
         rsp = self.send_message(req)
-        check_completion_code(rsp.completion_code)
+        check_completion_code(
+            rsp.completion_code,
+            req.cmdid,
```

Comment:
Is there a reason you do use the `req` here? Above you use `rsp`.

Reply:
On the other hand, would'nt it be better to have a new function like `check_rsp_completion_code(rsp)`?

Reply:
Wow that's my bad again, didn't sleep enough I guess. Sorry for this.

Yes I was asking myself the same thing but wasn't sure, if you think a function `check_rsp_completion_code(rsp)` comes handy too I will add it !

Reply:
Adding a new function it will reduce the linses when checking the cc and you only have to pass the rsp parameter.

Reply:
Apart from that it looks good!